Animations/Zoom/UpBig.js

```diff
@@ -26,5 +28,5 @@ var ZoomUpBig = {
     'transform': 'scale(.1) translateY(2000px)'
   }
 };
-//Use makeTransition function from main Tux Animation module to create wrapping animation componenet and pass in the default params
-module.exports = Animation.makeTransition('ZoomUpBig', ZoomUpBig);
+//Use makeAnimation function from main Tux Animation module to create wrapping animation componenet and pass in the default params
+module.exports = makeAnimation('ZoomUpBig', ZoomUpBig);
```

Comment:
This does not need to have `'ZoomUpBig'` passed into it since we have defined a `className` property in `ZoomUpBig`.